src/main/resources/application.properties

```diff
@@ -1 +1,13 @@
 
+# Database configuration props
+spring.datasource.url=jdbc:mysql://localhost:3306/demo?createDatabaseIfNotExist=true
+spring.datasource.username=root
+spring.datasource.password=secret
+spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
+
+
+# Hibernate props
+spring.jpa.show-sql=true
```

Comment:
Do we intend to use hibernate for main db connection too or only in local or testing ?

Reply:
yes